components/webapp-mgt/org.wso2.carbon.webapp.mgt/src/main/java/org/wso2/carbon/webapp/mgt/WebappAdmin.java

```diff
@@ -177,11 +178,38 @@ public WebappMetadata getStoppedWebapp(String webappFileName, String hostName) {
         return webappMetadata;
     }
 
+    //TODO WSAS-2125
+    private void setServiceListPath(WebApplication webApplication) {
+        String serviceListPathParamName = "service-list-path";
+        String serviceListPathParam =
+                webApplication.getContext().getServletContext().getInitParameter(serviceListPathParamName);
+        if (serviceListPathParam == null || "".equals(serviceListPathParam)) {
+            Map<String, ? extends ServletRegistration> servletRegs =
+                    webApplication.getContext().getServletContext().getServletRegistrations();
+            for (ServletRegistration servletReg : servletRegs.values()) {
+                serviceListPathParam = servletReg.getInitParameter(serviceListPathParamName);
+                if (serviceListPathParam != null || !"".equals(serviceListPathParam) ) {
+                    break;
+                }
+            }
+        }
+        if (serviceListPathParam == null || "".equals(serviceListPathParam)) {
+            serviceListPathParam = "/services";
+        } else {
+            serviceListPathParam = "";
```

Comment:
If the serviceListPathParam is not NULL, then serviceListPathParam always becomes empty.